pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -30,6 +30,45 @@
 import pandas.util.testing as tm
 
 
+def assert_invalid_comparison(left, right, box):
```

Comment:
Is this datetime-specific? Or should (/could) it be moved to pandas.util.testing?

Reply:
Potentially (there are a few assertion helper methods in this directory that might).  The main drawback is that if you pass a Series or DataFrame it only works with default name/index/columns

Reply:
can you add a doc-string / types